cypress/integration/landing.spec.js

```diff
@@ -0,0 +1,17 @@
+import { cyan } from "colors";
+
+describe('Landing Page', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+
+    it('Renders login page', () => {
+        cy.wait(5000);
+        cy.get('.ant-tabs-nav-list').should('be.visible');
+        cy.get('.ant-tabs-tab-btn').first().should('have.text', 'Real Estate');
```

Comment:
use const for cy.get('.ant-tabs-tab-btn')

Reply:
nope, doesn't work like that, will handle it separately